docs/api/non_geo_datasets.csv

```diff
@@ -1,33 +1,34 @@
-Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
```

Comment:
What changed in this file? I would expect only a 1-line change

Reply:
Line endings were Windows style, fixed